tools/cxbe/.gitignore

```diff
@@ -1 +1,2 @@
 cxbe
+dumpxbe
```

Comment:
nit: CXBE is kept rather minimalistic for nxdk (intentionally maybe?) to handle EXE > XBE conversion.

FYI, there's also https://github.com/XboxDev/xbedump which might be more capable.

Reply:
Neat, I didn't know that existed.

Is there any reason to keep the Xbe read constructor code in cxbe at all since it's otherwise not used and xbedump looks like it provides the same functionality?

Reply:
> FYI, there's also https://github.com/XboxDev/xbedump which might be more capable.

Agree here. We have multiple tools to dump XBE info already. Let's avoid another

> Is there any reason to keep the Xbe read constructor code in cxbe at all since it's otherwise not used and xbedump looks like it provides the same functionality?

Probably not. This is a remnant of original XBE&rarr;EXE conversion.

Reply:
Cool, will close this and send a PR to delete the dead code.